honeybee_ies/reader.py

```diff
@@ -469,11 +469,17 @@ def _parse_gem_segment(
         return faces
 
 
-def model_from_ies(gem: str) -> Model:
-    """Create a Honeybee Model from a VE GEM file."""
-    gem_file = pathlib.Path(gem)
-    # split the gem file into separate segments
-    segments = gem_file.read_text().split('\nLAYER')[1:]
+def model_from_gem(gem_str: str, model_id: str='Unnamed', model_name=None) -> Model:
+    """Create a Honeybee Model from the string contents of a VE GEM file.
+
+    Args:
+        gem_str: Text string representation of the contents of a GEM file.
+
```

Comment:
Now that we are adding docstrings we might want to add it for the other two arguments too.